tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -2414,6 +2414,29 @@ describe('metrics reducers', () => {
     });
   });
 
+  it('loads Step Selector Setting into the next state', () => {
```

Comment:
I'd prefer three different tests so we can test each property in isolation. Partially to maintain consistency with the rest of the file. But also to make sure, for example, that stepSelectorEnabled is not enabled by the linkedTimeEnabled setting in local storage.

Reply:
Separated.